Corgibytes.Freshli.Cli/Commands/CacheCommand.cs

```diff
@@ -1,35 +1,39 @@
 using System.CommandLine;
-using System.CommandLine.Invocation;
-using System.CommandLine.NamingConventionBinder;
 using Corgibytes.Freshli.Cli.CommandOptions;
-using Corgibytes.Freshli.Cli.CommandRunners;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Hosting;
 
 namespace Corgibytes.Freshli.Cli.Commands
 {
     public class CacheCommand : Command
     {
-        public CacheCommand() : base("cache", "Manages the local cache database")
+        public CacheCommand()
+            : base("cache", "Manages the local cache database")
         {
             CachePrepareCommand prepare = new();
             AddCommand(prepare);
+
+            CacheDestroyCommand destroy = new();
+            AddCommand(destroy);
```

Comment:
These variables can be inlined:

```suggestion
            AddCommand(new CachePrepareCommand());
            AddCommand(new CacheDestroyCommand());
```